packages/app/src/components/SlideWizard.tsx

```diff
@@ -165,7 +165,7 @@ export const SlideWizard = forwardRef(
               color={isError ? 'danger' : 'primary'}
             />
           </Stack>
-          <AnimatedStack pos="relative" style={animatedStyles} jc="space-between" fg={1}>
+          <AnimatedStack style={animatedStyles} flex={1} jc="space-between">
```

Comment:
Is this intentional, and does this not break things on other screens?

Reply:
Ah good question. this is of course not only used in this page. But yes it's intentional. If the content was longer than the screen the screen would overflow for the length of the component above. But will check if this doesn't break anything on the other screens.